packages/backend/src/graphql/schema.graphql

```diff
@@ -1590,32 +1590,7 @@ type Tenant implements Model {
   "The date and time that this tenant was deleted."
   deletedAt: String
   "List of settings for the tenant."
-  settings(
-    "Forward pagination: Cursor (wallet address key ID) to start retrieving settings after this point."
-    after: String
-    "Backward pagination: Cursor (wallet address key ID) to start retrieving keys before this point."
-    before: String
-    "Forward pagination: Limit the result to the first **n** keys after the `after` cursor."
-    first: Int
-    "Backward pagination: Limit the result to the last **n** keys before the `before` cursor."
-    last: Int
-    "Specify the sort order of keys based on their creation data, either ascending or descending."
-    sortOrder: SortOrder
-  ): TenantSettingsConnection
-}
-
-type TenantSettingsConnection {
-  "Information to aid in pagination."
-  pageInfo: PageInfo!
-  "A list of edges representing tenant settings and cursors for pagination."
-  edges: [TenantSettingEdge!]!
-}
-
-type TenantSettingEdge {
-  "A tenant setting node in the list."
-  node: TenantSetting!
-  "A cursor for paginating through the tenants."
-  cursor: String!
+  settings: [TenantSetting!]
```

Comment:
```suggestion
  settings: [TenantSetting!]!
```

Reply:
I am just thinking what I didn't do this, because I remember that initially I did exactly like this, but somehow I cannot remember why :/ But sure, let's try again :)